src/middlewares/auth.js

```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
```

Comment:
@gmemmy I think this should be called on the entry file `src/index.js`.
If it is called on the entry file, we wouldn't have to call it again, in so many other places.